src/SourceBuild/tarball/content/Directory.Build.props

```diff
@@ -216,13 +216,13 @@
     <!-- we don't produce the Windows version of this package but that's the one core-sdk keys off of for the ASP.NET version -->
     <ExtraPackageVersionPropsPackageInfo Include="MicrosoftAspNetCoreAppRuntimewinx64PackageVersion" Version="$(aspnetcoreOutputPackageVersion)" />
     <!-- same thing here for CLI -->
-    <ExtraPackageVersionPropsPackageInfo Include="MicrosoftNETCoreAppRuntimewinx64Version" Version="$(runtimeOutputPackageVersion)" />
+    <ExtraPackageVersionPropsPackageInfo Include="MicrosoftNETCoreAppRuntimewinx64Version" Version="%24(MicrosoftNETCoreAppRuntimeCentos8X64PackageVersion)" />
```

Comment:
Isn't this going to be an issue for distro maintainers?  We shouldn't be hard coding the rid here.  It looks like this was an issue already for https://github.com/dotnet/installer/pull/15121/files#diff-05d2bee19f4bffb22f9e29fa6f20386be04090ceeb1e8943cd9c71373b9d0800R241.  I am guessing that is alright because it is only for bootstrapping.

Do we need to compute the rid or is there a non-rid property we can use instead?

Reply:
Yes, we already depend on the CentOS 8 RID just a couple lines down.  Logged https://github.com/dotnet/source-build/issues/3231 to track this.